babel.config.js

```diff
@@ -0,0 +1,23 @@
+const { version } = require('./package.json')
+module.exports = {
+  presets: ['cozy-app'],
+  env: {
+    transpilation: {
+      plugins: [
+        [
+          'css-modules-transform',
+          {
+            extensions: ['.styl'],
+            preprocessCss: './preprocess',
+            extractCss: './transpiled/react/stylesheet.css',
+            generateScopedName: `[name]__[local]___cozy_ui_${version.replace(
+              '.',
+              '_'
+            )}`
```

Comment:
Qu'est ce que ca apporte d'avoir la version de `cozy-ui` et d'enlever le hash ? J'ai peut être pas compris le but.

Reply:
With the hash we will have different classes at each imports, like:
```
      .u-coolGrey {
      .u-coolGrey--12da1 {
      .u-coolGrey--1U0_S {
      .u-coolGrey--2flc7 {
       .u-coolGrey--2G1pW {
      .u-coolGrey--2r5Fs {
      .u-coolGrey--2VhFP {
      .u-coolGrey--2vQlR {
      .u-coolGrey--2xPIP {
      .u-coolGrey--3HU7I {
      .u-coolGrey--758MC {
      .u-coolGrey--jkmeV {
      .u-coolGrey--MlX_r {
      .u-coolGrey--Sr1MO {
```
Using a know id here will allow postcss to optimize and remove duplicates correctly

Reply:
The usual way not to add a hash is to use `:global` and we already use it via the `global()` mixin. BUT, we have a bug in the usage of `global()` that deactivates the wrapping in `:global`. See https://github.com/cozy/cozy-ui/issues/838. I would prefer if we'd solve this problem. I do not think removing the hash is a good way to solve the problem since it does not solve the root of the problem. WDYT ?

Reply:
The last commits fix this to use correctly css-modules with stylus, the result is finally even better than using this fixed ID 👍